backend/app/models/user_info.py

```diff
@@ -5,7 +5,7 @@ class UserInfo(mg.EmbeddedDocument):
     contact_name = mg.StringField(required=True)
     contact_email = mg.StringField(required=True)
     contact_phone = mg.StringField()
-    role = mg.StringField(choices=["Admin", "Donor", "ASP"], required=True)
+    role = mg.StringField(choices=["Admin", "Donor", "ASP", "User"], required=True)
```

Comment:
just curious - why do we need the additional "User" role here?

Reply:
Yes, it seems to me like we should stick with just the three of Admin, ASP, and Donor.